ml.q

```diff
@@ -689,9 +689,9 @@ rt:dt[oig;oig;wmse]             / regression tree
 
 / generate (n) decision trees by applying (f) to a resampled (with
 / replacemnt) (t)able
-bag:{[n;f;t](f ?[;t]::) peach n#count t} / Bootstrap AGgregating
+bag:{[n;f;t](f ?[;t]::) peach n#count t} / Bootstrap Aggregating
 
-/ given an atom or list (k), and Bootstrap AGgregating (m)odel, make
```

Comment:
you are not the first to comment on this. Bootstratp AGgregating was explicit to indicate BAG. but better to make the whole thing lower-case to prevent confusion